wolfcrypt/test/test.c

```diff
@@ -4346,6 +4365,18 @@ int hc128_test(void)
 
     int times = sizeof(test_hc128) / sizeof(testVector), i;
 
+    int ret = 0;
+#ifdef WOLFSSL_NO_MALLOC
```

Comment:
Maybe account for small stack. Use `!defined(WOLFSSL_SMALL_STACK) || defined(WOLFSSL_NO_MALLOC)`

Reply:
done.  this is the sort of refactor I proposed above for dh_test() and whatnot, but here it's a one-liner so a no-brainer to go ahead with it.  maybe those others will be, or can be, one-liners too.  I'll look at it.